projects/storefrontlib/src/lib/store-finder/components/store-finder-search-result/store-finder-list/store-finder-list.component.spec.ts

```diff
@@ -36,7 +38,7 @@ class GoogleMapRendererServiceMock {
 describe('StoreFinderDisplayListComponent', () => {
   let component: StoreFinderListComponent;
   let fixture: ComponentFixture<StoreFinderListComponent>;
-  let store: Store<fromReducers.StoresState>;
+  let store: Store<StoresState>;
```

Comment:
remove 'store' and mock all used services.

Reply:
Done.